test/DADBridgeWrapper.ts

```diff
@@ -0,0 +1,163 @@
+import Contracts from '../components/Contracts';
+import { TestERC20Token } from '../typechain';
+import { DADBridgeWrapper } from '../typechain';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { expect } from 'chai';
+import { utils, BigNumber } from 'ethers';
+import { ethers } from 'hardhat';
+
+const ROLE_ADMIN = utils.id('ROLE_ADMIN');
+const TOTAL_SUPPLY = BigNumber.from(1_000_000_000);
+const MINT_AMOUNT = BigNumber.from(1_000_000);
+const INVALID_TOKEN = '0x'.padEnd(42, '0');
+const INVALID_RECIPIENT = '0x'.padEnd(42, '0');
+const INVALID_AMOUNT = BigNumber.from(0);
+
+describe('DADBridgeWrapper', () => {
+    let deployer: SignerWithAddress;
+    let admin: SignerWithAddress;
+    let user: SignerWithAddress;
+
+    let token: TestERC20Token;
+    let dadBridgeWrapper: DADBridgeWrapper;
+
+    const assertState = async (contractBalance: BigNumber, userBalance: BigNumber) => {
+        expect(await dadBridgeWrapper.totalSupply()).to.equal(userBalance);
+        expect(await token.balanceOf(user.address)).to.equal(userBalance);
+        expect(await token.balanceOf(dadBridgeWrapper.address)).to.equal(contractBalance);
+    };
+
+    before(async () => {
+        [deployer, admin, user] = await ethers.getSigners();
+    });
+
+    describe('Constructions:', () => {
```

Comment:
We don't title our tests this way (capital letter + `:`)

Reply:
Fixed.